deploy.rb

```diff
@@ -105,6 +121,12 @@ def comment_out_branch_prompt!
 mode_display = mode ? mode.gsub('--', '') : 'deploy'
 puts "repos to #{mode_display}: #{repo_names.join(', ')}"
 
+
+unless ssh_check || check_cocina
+  # Runs cocina check before deployment
+  exit unless cocina_check
+end
```

Comment:
seems like it'd be better to run this after the repos get updated.  will throw up a follow-on PR with what i'm imagining in a moment...

Reply:
ok, follow on PR: #54

if you like that and decide to merge it, we should prob get someone else to merge this to `main`, since we essentially paired asynchronously on it.